frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts

```diff
@@ -92,6 +106,11 @@ if (clusterStatus && cephHealth) {
       await verifyFields();
       await click(confirmButton);
 
+      expansionObjects.osdTree = JSON.parse(
+        execSync(
+          `oc -n ${NS} rsh  $(oc -n ${NS} get pod | grep ceph-operator| awk '{print$1}') ceph --conf=/var/lib/rook/${NS}/${NS}.config osd tree --format=json`,
+        ).toString(),
+      );
       const statusCol = storageClusterRow(uid).$('td:nth-child(4)');
```

Comment:
This is not reliable, in 4.4 the status is in 3 column.
Use something else.

Reply:
Don't see any other option. This test is written w.r.t 4.3 setup.

Reply:
So, we are okay with breaking tests in 4.4+. This will cause timeout error, because it will never match "ready" status. I am not into integration testing but may be adding ids (though its touching core). There would be some way may be xpath ?

Reply:
My point of view to these inter-version changes in the UI is that once a PR is directed to console/master, it should be working in console/master.
I know it's hard to maintain upstream tests working, especially without a gating job that would enforce developers to fix tests that their changes break.

But if we implement test cases intentionally for old versions, it inevitably means that more time will be invested (or more like wasted, as it's not really an investment) into fixing the test for the next version.

Reply:
Agree with @rhazdil , as tests are not backported to 4.3/4.2 due to code freeze.
Hence it makes sense to write tests against master now and follow this suggested way to maintain.

Reply:
@afreen23 with 4.4 setup as well, the status is in 4th column only. Can you share the screenshot, where you have seen it in 3rd column?

Reply:
For adding class to this status field, require changes in the common components i.e. `OperandTableHeader` which is being used by all the `provided APIs` views. Hence, I don't think that its a good idea to modify the common component just for adding class, as I am not sure, what impacts it might have.